sources/PTYSession.m

```diff
@@ -223,6 +223,9 @@ @implementation PTYSession {
     // The code to send in the anti idle timer.
     char _antiIdleCode;
 
+    // The interval between sending anti-idle codes.
+    NSTimeInterval _antiIdlePeriod;
```

Comment:
It's not necessary to declare an ivar named `_antiIdlePeriod` when there is a property named `antiIdlePeriod`. It will be created automatically (this is called *automatic property synthesis*). Looks like you followed the model of `antiIdleCode`, which actually predates this language feature. Please remove both this and _antiIdleCode.